### ISSUE TITLE
Customize: display_options: icon

### ISSUE THREAD

**Issue by isablend** [nielsfaber/scheduler-card#395]
For defined actions, I've identified associated icons (for example an open window for a window entity, where the action is a script to open the window, along with some other things), 
      customize:
         script.vent_cinema_open:
           name: "Cinema Vent"
           icon: window-closed
           actions:
             - service: script.vent_cinema_open
               name: "Open"
               icon: window-open

but when the schedule summary display shows the icon doesn't show the icon associated with the action that has been selected, but rather the action which is to run a script.
I've tried the display_options: icon: "action" and display_options: icon: "entity", but neither displays the icon i want associated with the scheduled activity.

The use case I want is to show the icon defined in the 'actions' definition.   Can this be done?

**Comment by nielsfaber** [comment]
Could you update the scheduler-card to the latest version and try again?
There were some troubles in the last release, maybe your problems are related to it.

**Comment by isablend** [comment]
I've updated both the scheduler and the scheduler card to the last available version, but I don't think its made any difference.
I have the following customisation for some esphome plugs.
  switch.music_amp_switch: #8
    friendly_name: Music Amp
    device_class: plug
    icon: mdi:amplifier
  switch.bug_zapper_switch: #9
    friendly_name: Bug Zapper 
    device_class: plug
    icon: mdi:bug

both icons in the schedule card are defined as
actions:
  - service: turn_on
    name: "On"
    icon: toggle-switch-outline
  - service: turn_off
    name "Off"
    icon: toggle-switch-off
 
with

display_options:
  icon: "entity"

the bug zapper is being displayed as a folder icon and the music amp is being correctly displayed.

with display_options:
  icon: "action"

the bug zapper is being displayed as a folder icon.

in both cases the scheduled action is being displayed as a power icon.

**Comment by nielsfaber** [comment]
I cannot reproduce your problem.
With `display_options: {icon: entity}`:
![image](https://user-images.githubusercontent.com/16861605/118307255-b6f83980-b4ea-11eb-8b84-096b38b45951.png)
After adding:
```
customize:
  switch.switch_2:
    icon: hass:bug
```
![image](https://user-images.githubusercontent.com/16861605/118307277-c2e3fb80-b4ea-11eb-8cee-78c3ddebd3bc.png)

With `display_options: {icon: action}`:
![image](https://user-images.githubusercontent.com/16861605/118307325-d3947180-b4ea-11eb-9345-6df2ec916e42.png)

After adding:
```
switch.switch_2:
  actions:
    - service: turn_off
      icon: hass:toggle-switch-off
```

![image](https://user-images.githubusercontent.com/16861605/118307409-f161d680-b4ea-11eb-8cec-11b9da4cbe3a.png)


What am I missing? 😅 

Perhaps you need to refresh your browser cache after updating? Please make sure you're running:
![image](https://user-images.githubusercontent.com/16861605/118307610-37b73580-b4eb-11eb-9945-f578427e095b.png)

**Comment by isablend** [comment]
I've checked the version and that is as you say 2.1.3, so we're good there.
I went back and specified the icon's in the way you have in your example and it works exactly as you say.   The difference with my initial setup is that I have not stated the icon in the customize section of the card.   I've done it in the customize section of configuration.yaml.   Now for the majority of entities icons specified in that way they roll-through.   So the card isn't inheriting the icon definition from the entity customization, but expecting it from the card definition.

In the case with the icon coming from the 'action' It seems that specifying icon: hass:bug for the action, rather than icon: mdi:bug seems to do the trick.
